io/src/lib.rs

```diff
@@ -0,0 +1,174 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
```

Comment:
```suggestion
use gstd::{prelude::*, ActorId};
```
These traits have already been imported by `gstd::prelude::*`.